modules/user_accounts/php/edit_user.class.inc

```diff
@@ -598,7 +598,7 @@ class Edit_User extends \NDB_Form
                 'UserID_Group',
                 'User name',
                 $this->_GUIDelimiter,
-                false
+                array()
```

Comment:
If the new declaration of `addGroup` will default the 5th parameter to an empty array, why include an empty here?

Reply:
I changed the calling code to fix some bugs that came up during testing, but you're right they now don't need to be added. I'll remove the 5th param call.